generated/nirfmxinstr/nirfmxinstr.proto

```diff
@@ -175,12 +175,12 @@ enum NiRFmxInstrAttribute {
   NIRFMXINSTR_ATTRIBUTE_COMMON_MODE_LEVEL = 70;
   NIRFMXINSTR_ATTRIBUTE_SMU_RESOURCE_NAME = 71;
   NIRFMXINSTR_ATTRIBUTE_SMU_CHANNEL = 72;
-  NIRFMXINSTR_ATTRIBUTE_AUTOMATIC_SG_SA_SHARED_LO = 74;
+  NIRFMXINSTR_ATTRIBUTE_AUTOMATIC_SG_SA_SHARED_LO_ENABLED = 74;
   NIRFMXINSTR_ATTRIBUTE_OVERFLOW_ERROR_REPORTING = 77;
   NIRFMXINSTR_ATTRIBUTE_LO_IN_POWER = 78;
   NIRFMXINSTR_ATTRIBUTE_LO_OUT_POWER = 79;
   NIRFMXINSTR_ATTRIBUTE_LO_VCO_FREQUENCY_STEP_SIZE = 80;
-  NIRFMXINSTR_ATTRIBUTE_LO_PLL_FRACTIONAL_MODE = 90;
+  NIRFMXINSTR_ATTRIBUTE_LO_PLL_FRACTIONAL_MODE_ENABLED = 90;
```

Comment:
Where these just unsynced changes? They didnt come from your scrapigen change

Reply:
Yes, they came from [_my_ scrapigen change](https://github.com/ni/grpc-device-scrapigen/commit/e2b54bf65cbbea12820cf2b9fbd416848d230c0f), which I apparently neglected to copy over.